src/components/UserDialog.tsx

```
@@ -7,114 +7,181 @@ import {
+        [field]: value,
+      }))
+    }
+  }
+
+  const handleSubmit = async (e: React.FormEvent) => {
     e.preventDefault()
-    onUpdate(formData)
+
+    const cleanData = Object.fromEntries(
+      Object.entries(formData).filter(
+        ([_, value]) => value !== '' && value !== null && value !== undefined
```

ESLint

[error] '_' is defined but never used.

(@typescript-eslint/no-unused-vars)